app/pods/components/hls-player/component.js

```diff
@@ -46,13 +44,22 @@ export default Component.extend(KeyboardShortcuts, {
     if (this.copySrc === this.src)
       return ;
 
+    function queryParams (source) {
+      let array = [];
+      for(let key in source) {
```

Comment:
use `.map()` instead of for loop

Reply:
`source` is an object, map won't work here.